components/collective-page/sections/Events.js

```diff
@@ -62,50 +63,42 @@ class SectionEvents extends React.PureComponent {
     return (
       <Box pb={4}>
         <HorizontalScroller getScrollDistance={this.getContributeCardsScrollDistance}>
-          {(ref, Chevrons) => (
-            <div>
-              <ContainerSectionContent pb={3}>
-                <Flex justifyContent="space-between" alignItems="center">
-                  <H3 fontSize={['20px', '24px', '32px']} fontWeight="normal" color="black.700">
-                    <FormattedMessage id="Events" defaultMessage="Events" />
-                  </H3>
-                  <Box m={2} flex="0 0 50px">
-                    <Chevrons />
-                  </Box>
-                </Flex>
-              </ContainerSectionContent>
-
-              <ContributeCardsContainer ref={ref}>
-                {isAdmin && (
-                  <Box px={CONTRIBUTE_CARD_PADDING_X} minHeight={150}>
-                    <CreateNew route={`/${collective.slug}/events/create`} data-cy="create-event">
-                      <FormattedMessage id="event.create.btn" defaultMessage="Create Event" />
-                    </CreateNew>
-                  </Box>
-                )}
-                {upcomingEvents.map(event => (
-                  <Box key={event.id} px={CONTRIBUTE_CARD_PADDING_X}>
-                    <ContributeEvent
-                      collective={collective}
-                      event={event}
-                      hideContributors={hasNoContributorForEvents}
-                      disableCTA={!collective.isActive || !event.isActive}
-                    />
-                  </Box>
-                ))}
-                {pastEvents.map(event => (
-                  <Box key={event.id} px={CONTRIBUTE_CARD_PADDING_X}>
-                    <ContributeEvent
-                      collective={collective}
-                      event={event}
-                      hideContributors={hasNoContributorForEvents}
-                      disableCTA={!collective.isActive || !event.isActive}
-                    />
-                  </Box>
-                ))}
-              </ContributeCardsContainer>
-            </div>
-          )}
+          <div>
+            <Box pb={3} px={Dimensions.PADDING_X}>
+              <H3 fontSize={['20px', '24px', '32px']} fontWeight="normal" color="black.700">
+                <FormattedMessage id="Events" defaultMessage="Events" />
+              </H3>
+            </Box>
+            <ContributeCardsContainer>
+              {isAdmin && (
+                <Box px={CONTRIBUTE_CARD_PADDING_X} minHeight={150}>
+                  <CreateNew route={`/${collective.slug}/events/create`} data-cy="create-event">
+                    <FormattedMessage id="event.create.btn" defaultMessage="Create Event" />
+                  </CreateNew>
+                </Box>
+              )}
+              {upcomingEvents.map(event => (
```

Comment:
I think we could simplify this duplication of identical `<Box>` elements by, 🤔 😄 

```
{[...upcomingEvents, ...pastEvents].map(event => (
    <Box key={...
```

Reply:
Yeah, lets do it 🚀